cfme/infrastructure/config_management.py

```diff
@@ -463,96 +705,104 @@ def step(self):
 
     def resetter(self):
         if self.obj.appliance.version >= '5.8' and self.obj.type == 'Ansible Tower':
-            accordion.tree('Providers', 'All Ansible Tower Providers')
-        else:
-            accordion.tree('Providers', 'All Configuration Manager Providers')
-        tb.select('Grid View')
-
-    def am_i_here(self):
-        if self.obj.appliance.version >= '5.8' and self.obj.type == 'Ansible Tower':
-            page = 'All Ansible Tower Providers'
+            self.view.sidebar.providers.tree.click_path('All Ansible Tower Providers')
         else:
-            page = 'All Configuration Manager Providers'
-        return match_page(summary=page)
+            self.view.sidebar.providers.tree.click_path('All Configuration Manager Providers')
+        self.view.toolbar.view_selector.select('Grid View')
```

Comment:
**REQUIRED**
Here and below. You shouldn't reset the view to "Grid View", use `BaseEntitiesView` to represent different entities.